ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1059,10 +1117,20 @@ export const RuntimePanel = fp.flow(
                 <div>You've made changes that require recreating your environment to take effect.</div>
              </WarningMessage>
            }
-           {(errors || dataprocErrors) &&
-             <WarningMessage>
-               {getErrorsTooltipContent()}
-             </WarningMessage>
+           {
+             ((
+               workspace.billingAccountType === BillingAccountType.FREETIER && (freeTierErrors || freeTierDataprocErrors)
+             ) || (
+               workspace.billingAccountType === BillingAccountType.USERPROVIDED && (paidTierErrors || paidTierDataprocErrors)
```

Comment:
can we just check if `getErrorMessageContent()` is non empty and render if so?

Reply:
yeah this could be cleaned up a bit